README.md

```diff
@@ -1,2 +1 @@
-# review-your-own-trial
-Excercise of review my own trial to better understand PR reviews.
+# microverse_trial_2
```

Comment:
Readme file is completely empty.

Checkout [Microverse Repo guide](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) on how to make your repo and README.md file look professional.

Reply:
- [ ] A good practice is to have a well-documented README file. This file is important so people checking your repository know what it is about. You can also include information about the author, the project, technologies used, etc.

Check out this [Microverse Repo guide](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) on how to make your repo and README.md file look professional.